internal/controllers/module_nmc_reconciler.go

```diff
@@ -357,7 +362,7 @@ func (mnrh *moduleNMCReconcilerHelper) enableModuleOnNode(ctx context.Context, m
 	}
 
 	opRes, err := controllerutil.CreateOrPatch(ctx, mnrh.client, nmcObj, func() error {
-		err = mnrh.nmcHelper.SetModuleConfig(nmcObj, mld, &moduleConfig)
+		err := mnrh.nmcHelper.SetModuleConfig(nmcObj, mld, &moduleConfig)
```

Comment:
Should we perhaps put this line in the `if` statement?

```suggestion
		if err := mnrh.nmcHelper.SetModuleConfig(nmcObj, mld, &moduleConfig); err != nil {
```

Reply:
I would, but this is @yevgeny-shnaidman's style and I did not want to touch outside code for clarity.

Reply:
No worries then, let's leave it like that. I didn't pay attention to the existing style, but I should have :)